test/docker-compose.integration-tests.yml

```diff
@@ -3,15 +3,16 @@ version: "3.8"
 # environment variable PWD is assumed to be grapl root directory
 
 services:
-  rust-node-identifier-integration-tests:
+
+  rust-integration-tests:
     image: grapl/rust-integration-tests:${TAG:-latest}
     build:
       context: ${PWD}/src
       dockerfile: rust/Dockerfile
       target: build-test
       args:
         - RUST_BUILD=test-integration
-    command: cargo test --features node-identifier/integration --test '*'
+    command: cargo test --features sqs-executor/integration,node-identifier/integration,kafka-metrics-exporter/integration --test '*'
```

Comment:
- Can you just add a comment in <both places that specify `--features`> to update the other one? like, comment here to update the dockerfile + comment in dockerfile to update it here
- Not a must have, but thoughts on baking this into a CMD in the dockerfile instead? separate command made sense when multiple integration tests were using the same image, but that seems no longer the case

Reply:
Good idea. I added a diff to use the default command in the Dockerfile to bring that value to one file, and then I move it to a variable so there's only one spot now. 👍